keyboards/quantrik/kyuu/info.json

```diff
@@ -0,0 +1,82 @@
+          {"label":"K0C (B6,D2)", "x":12, "y":0},
+          {"label":"K0D (B6,D1)", "x":13, "y":0},
+          {"label":"K0E (B6,B3)", "x":14, "y":0},
+          {"label":"K2E (B4,B3)", "x":15, "y":0},
+          {"label":"K10 (B5,F1)", "x":0, "y":1, "w":1.5},
+          {"label":"K11 (B5,F4)", "x":1.5, "y":1},
+          {"label":"K12 (B5,F5)", "x":2.5, "y":1},
+          {"label":"K13 (B5,F6)", "x":3.5, "y":1},
+          {"label":"K14 (B5,F7)", "x":4.5, "y":1},
+          {"label":"K15 (B5,C7)", "x":5.5, "y":1},
+          {"label":"K16 (B5,C6)", "x":6.5, "y":1},
+          {"label":"K17 (B5,F0)", "x":7.5, "y":1},
+          {"label":"K18 (B5,B7)", "x":8.5, "y":1},
+          {"label":"K19 (B5,D0)", "x":9.5, "y":1},
+          {"label":"K1A (B5,D5)", "x":10.5, "y":1},
+          {"label":"K1B (B5,D3)", "x":11.5, "y":1},
+          {"label":"K1C (B5,D2)", "x":12.5, "y":1},
+          {"label":"K1D (B5,D1)", "x":13.5, "y":1, "w":1.5},
+          {"label":"K1E (B5,B3)", "x":15, "y":1},
+          {"label":"K20 (B4,F1)", "x":0, "y":2, "w":1.75},
+          {"label":"K21 (B4,F4)", "x":1.75, "y":2},
+          {"label":"K22 (B4,F5)", "x":2.75, "y":2},
+          {"label":"K23 (B4,F6)", "x":3.75, "y":2},
+          {"label":"K24 (B4,F7)", "x":4.75, "y":2},
+          {"label":"K25 (B4,C7)", "x":5.75, "y":2},
+          {"label":"K26 (B4,C6)", "x":6.75, "y":2},
+          {"label":"K27 (B4,F0)", "x":7.75, "y":2},
+          {"label":"K28 (B4,B7)", "x":8.75, "y":2},
+          {"label":"K29 (B4,D0)", "x":9.75, "y":2},
+          {"label":"K2A (B4,D5)", "x":10.75, "y":2},
+          {"label":"K2B (B4,D3)", "x":11.75, "y":2},
+          {"label":"K2C (B4,D2)", "x":12.75, "y":2, "w":2.25},
+          {"label":"K2D (B4,D1)", "x":15, "y":2},
+          {"label":"K30 (D7,F1)", "x":0, "y":3, "w":2.25},
+          {"label":"K31 (D7,F4)", "x":2.25, "y":3},
+          {"label":"K32 (D7,F5)", "x":3.25, "y":3},
+          {"label":"K33 (D7,F6)", "x":4.25, "y":3},
+          {"label":"K34 (D7,F7)", "x":5.25, "y":3},
+          {"label":"K35 (D7,C7)", "x":6.25, "y":3},
+          {"label":"K36 (D7,C6)", "x":7.25, "y":3},
+          {"label":"K37 (D7,F0)", "x":8.25, "y":3},
+          {"label":"K38 (D7,B7)", "x":9.25, "y":3},
+          {"label":"K39 (D7,D0)", "x":10.25, "y":3},
+          {"label":"K3A (D7,D5)", "x":11.25, "y":3},
+          {"label":"K3B (D7,D3)", "x":12.25, "y":3, "w":1.75},
+          {"label":"K3C (D7,D2)", "x":14, "y":3},
+          {"label":"K3D (D7,D1)", "x":15, "y":3},
+          {"label":"K40 (D6,F1)", "x":0, "y":4, "w":1.5},
+          {"label":"K41 (D6,F4)", "x":1.5, "y":4},
+          {"label":"K42 (D6,F5)", "x":2.5, "y":4, "w":1.5},
+          {"label":"K46 (D6,C6)", "x":4, "y":4, "w":7},
+          {"label":"K4A (D6,D5)", "x":11, "y":4, "w":1.5},
+          {"label":"K4B (D6,D3)", "x":13, "y":4},
+          {"label":"K4C (D6,D2)", "x":14, "y":4},
+          {"label":"K4D (D6,D1)", "x":15, "y":4}
+        ]
+      }
+    }
+  }
+  
```

Comment:
The indentation here looks a bit wonky. The first level is 4 spaces while the rest are 2. You could make it consistent by deleting the first two columns on lines 2–81 (i.e. reducing the indentation by 2 spaces).

Reply:
you'll have to take that up with noroadsleft.

Reply:
My parser uses 2-space indent for `info.json`. It probably got modified by the IDE, .editorconfig or something similar.